### PR TITLE
Fix [CON-203] - Color of arrowheads does not match with color of lines

### PR DESCRIPTION
**Pull request by habemuscode**
**Issue**

https://oat-sa.atlassian.net/browse/CON-203

**Related work**

https://github.com/oat-sa/extension-tao-itemqti/issues/1577

**Description**

The color of all arrowheads (including axis arrowheads) changes, when we change the current line or add new. So, it will always inherit the color from the last path created with the arrow-end/ arrow-start attribute set.
 
imagen

The full description of [issue](https://github.com/oat-sa/extension-tao-itemqti)/issues/1577

Also, it looks like the PCI settings are broken on the construct side. When trying to open Lines options an error pops up in the console.

imagen

**AC**

- [x] Fix arrow head color
- [ ] Fix error in console of PCI settings

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -53,7 +53,7 @@
     'label' => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license' => 'GPL-2.0',
-    'version' => '40.3.12',
+    'version' => '40.3.13',
```
Comment (mike-ionut-mihai-sandu-tao) [inline manifest.php:56]: why ?

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]

File: manifest.php (hunk 1 above)
Comment (habemuscode) [reply manifest.php:56]: Already pushed npm dependencies update

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by 404passionFruit — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**